### DISCUSSION TITLE
Welcome to fozzie-components Discussions!

### DISCUSSION THREAD

**Discussion by ashleynolan** [justeattakeaway/fozzie-components#524] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by tommcclean** [comment]
first
